infrastructure/oss/cli_scanner.go

```diff
@@ -238,7 +238,7 @@ func (cliScanner *CLIScanner) updateArgs(workDir types.FilePath, commandLineArgs
 	folderConfigArgs := folderConfig.AdditionalParameters
 
 	// this asks the client for the current SDK and blocks on it
-	additionalParameters := cliScanner.updateSDKs(workDir)
+	additionalParameters := cliScanner.updateSDKs(folderConfig.FolderPath)
```

Comment:
SDK config should always be taken from the actual IDE path, not the reference path.